sourcemash/frontend/static/js/views/items_view.js

```diff
@@ -52,6 +52,7 @@ Sourcemash.Views.ItemsView = Backbone.View.extend({
 
             var itemCardViewOneCol = new Sourcemash.Views.ItemCardView({el: "#item-" + item.get('id') + "-onecol", model: item, user: user });
             itemCards.push(itemCardViewOneCol);
+            $(".card-reveal a").attr("target","_blank");
```

Comment:
Yes, you are missing something. The problem is that we render item.summary as HTML and we want THOSE links to open in new tabs as well. So we need to either process that using beautiful soup on the backend (nontrivial - I tried it) or jquery on the frontend (the one line you see here).